src/endpoints/get-dacs/domain/get-dacs.controller.ts

```diff
@@ -0,0 +1,75 @@
+import { injectable, Result } from '@alien-worlds/api-core';
+import { AlienWorldsAccount } from '@alien-worlds/eosdac-api-common';
```

Comment:
I think you should use Account entity from the TokenWorlds not AlienWorlds.

AlienWorlds will be removed from eosdac-api

Reply:
game-ui is reading from alien.worlds contract in this case, so probably we need to keep this in eosdac-api.